pages/sdk/onramp-kit/guides/stripe.mdx

```diff
@@ -1,119 +1,152 @@
-import { Steps } from 'nextra/components'
+import { Steps, Callout } from 'nextra/components'
+import Image from 'next/image'
 
 # Integration with Stripe
 
-The [Stripe Crypto Onramp service](https://stripe.com/docs/crypto/overview) allows individuals to securely purchase cryptocurrencies from your application. 
+<Callout type="info" emoji="ℹ️">
```

Comment:
@germartinez might have thoughts on if we should remove this as he was working on a different style on redirection. Might be good to check this with him.

Reply:
@germartinez I will wait for our discussion to resolve this comment. I see three possibilities: 1. I remove the callout. 2. I make it optional. 3. I leave it as it is.

I am also happy to create an importable component that checks for a get parameter in the URL and displays a Callout conditionally.

Reply:
Resolved: We leave it like it is for this PR and create a new PR to make it conditional.